internal/controller/http/command.go

```diff
@@ -31,10 +33,17 @@ func (c *RestController) Command(writer http.ResponseWriter, request *http.Reque
 	var reserved url.Values
 	vars := mux.Vars(request)
 	correlationID := request.Header.Get(sdkCommon.CorrelationHeader)
+	if correlationID == "" {
+		correlationID = request.Context().Value(sdkCommon.CorrelationHeader).(string)
```

Comment:
type casting might cause runtime error.  please modify the code to

correlationID, _ = request.Context().Value(sdkCommon.CorrelationHeader).(string)